go.mod

```diff
@@ -12,16 +12,25 @@ require (
 	github.com/google/go-cmp v0.5.2
 	github.com/imdario/mergo v0.3.11 // indirect
 	github.com/mholt/archiver v2.1.0+incompatible
+	github.com/moby/term v0.0.0-20200915141129-7f0af18e79f2 // indirect
 	github.com/nwaples/rardecode v1.1.0 // indirect
 	github.com/prometheus/client_golang v1.8.0
 	github.com/spf13/afero v1.4.1
 	github.com/ulikunitz/xz v0.5.7 // indirect
 	gopkg.in/yaml.v2 v2.3.0
+	gotest.tools/v3 v3.0.3 // indirect
 	helm.sh/helm/v3 v3.3.4
 	k8s.io/apimachinery v0.18.9
 	k8s.io/client-go v0.18.9
 	rsc.io/letsencrypt v0.0.3 // indirect
 	sigs.k8s.io/controller-runtime v0.6.3
 )
 
-replace sigs.k8s.io/cluster-api => github.com/giantswarm/cluster-api v0.3.10-gs
+replace (
+	// Use moby v20.10.0-beta1 to fix build issue on darwin.
+	github.com/docker/docker => github.com/moby/moby v20.10.0-beta1+incompatible
+	// Use mergo 0.3.11 due to bug in 0.3.9 merging Go structs.
+	github.com/imdario/mergo => github.com/imdario/mergo v0.3.11
```

Comment:
@tomahawk28 I've moved this to a replace. As otherwise I worry we might miss reverting to the buggy version.

WDYT?

Reply:
Good, Keep both in `require` and `replace` would work.

Reply:
I just bumped the orb to the latest too. To get that out of the way. I'll merge once CI is green.

Reply:
Nancy finds a ton of stuff :( 

https://app.circleci.com/pipelines/github/giantswarm/helmclient/366/workflows/4ef17a1d-9972-4fe3-9a27-558e7ad40f50/jobs/3600

I've reverted and will fix it separately.